samples/sample_encode/src/sample_encode.cpp

```diff
@@ -1648,6 +1648,34 @@ mfxStatus ParseInputString(msdk_char* strInput[], mfxU8 nArgNum, sInputParams* p
     }
 #endif
 
+    mfxU16 mfxU16Limit = std::numeric_limits<mfxU16>::max();
+    if (pParams->MaxKbps > mfxU16Limit || pParams->nBitRate > mfxU16Limit ||
+        pParams->InitialDelayInKB > mfxU16Limit || pParams->BufferSizeInKB > mfxU16Limit)
+    {
+        mfxU32 maxVal = std::max<mfxU32>({ pParams->MaxKbps,
+                                           pParams->nBitRate,
+                                           pParams->InitialDelayInKB,
+                                           pParams->BufferSizeInKB });
+        pParams->nBitRateMultiplier += (mfxU16)std::ceil(static_cast<double>(maxVal) / mfxU16Limit);
```

Comment:
do we really need '+=' ?

Reply:
I think, yes, user can set own nBitRateMultiplier

Reply:
'yes, we need it' - for smt, but for sample_encode it is not needed and deleted